### ISSUE TITLE
Asking about z-score normalization

### ISSUE THREAD

**Issue by kietdoan** [pulp-bio/BioFoundation#5]
First of all, I would like to thanks the authors for sharing this work. I would like to ask that in your document, you state that you use z-score normalization per channel. However, in your code, I do not see where you apply this normalization. Could you point out you normalize in which steps?

Kind regards,

**Comment by Thoriri** [comment]
Hi @kietdoan, sorry for not getting back to you earlier. Berkay will answer this in a couple of days.

**Comment by berkaydoner** [comment]
Hi @kietdoan, you can find the channel normalization code in pretrain and finetuning tasks for LUNA:

https://github.com/pulp-bio/BioFoundation/blob/d88596590f3bd3fce573be07646b7d3977ce7bcc/tasks/pretrain_task_LUNA.py#L30-L39
